zhaquirks/inovelli/__init__.py

```diff
@@ -0,0 +1,206 @@
+            0x0048: ("Default_LED3_Strip_Intensity_When_On", t.uint8_t),
+            0x0049: ("Default_LED3_Strip_Intensity_When_Off", t.uint8_t),
+            0x004B: ("Default_LED4_Strip_Color_When_On", t.uint8_t),
+            0x004C: ("Default_LED4_Strip_Color_When_Off", t.uint8_t),
+            0x004D: ("Default_LED4_Strip_Intensity_When_On", t.uint8_t),
+            0x004E: ("Default_LED4_Strip_Intensity_When_Off", t.uint8_t),
+            0x0050: ("Default_LED5_Strip_Color_When_On", t.uint8_t),
+            0x0051: ("Default_LED5_Strip_Color_When_Off", t.uint8_t),
+            0x0052: ("Default_LED5_Strip_Intensity_When_On", t.uint8_t),
+            0x0053: ("Default_LED5_Strip_Intensity_When_Off", t.uint8_t),
+            0x0055: ("Default_LED6_Strip_Color_When_On", t.uint8_t),
+            0x0056: ("Default_LED6_Strip_Color_When_Off", t.uint8_t),
+            0x0057: ("Default_LED6_Strip_Intensity_When_On", t.uint8_t),
+            0x0058: ("Default_LED6_Strip_Intensity_When_Off", t.uint8_t),
+            0x005A: ("Default_LED7_Strip_Color_When_On", t.uint8_t),
+            0x005B: ("Default_LED7_Strip_Color_When_Off", t.uint8_t),
+            0x005C: ("Default_LED7_Strip_Intensity_When_On", t.uint8_t),
+            0x005D: ("Default_LED7_Strip_Intensity_When_Off", t.uint8_t),
+            0x0034: ("Smart_Bulb_Mode", t.Bool),
+            0x0035: ("Double_Tap_Up_for_Full_Brightness", t.Bool),
+            0x005F: ("LED_Color_When_On", t.uint8_t),
+            0x0060: ("LED_Color_When_Off", t.uint8_t),
+            0x0061: ("LED_Intensity_When_On", t.uint8_t),
+            0x0062: ("LED_Intensity_When_Off", t.uint8_t),
+            0x0100: ("Local_Protection", t.Bool),
+            0x0101: ("Remote_Protection", t.Bool),
+            0x0102: ("Output_Mode", t.Bool),
+            0x0103: ("On_Off_LED_Mode", t.Bool),
+            0x0104: ("Firmware_Progress_LED", t.Bool),
+        }
+    )
+
+    server_commands = {
+        0x00: ("button_event", (t.uint8_t, t.uint8_t), False),
+        0x01: (
+            "led_effect",
+            (t.uint8_t, t.uint8_t, t.uint8_t, t.uint8_t),
+            False,
+        ),  # LED Effect
+        0x03: (
+            "individual_led_effect",
+            (t.uint8_t, t.uint8_t, t.uint8_t, t.uint8_t, t.uint8_t),
+            False,
+        ),  # individual LED Effect
+    }
+
+    def handle_cluster_request(
+        self,
+        hdr: foundation.ZCLHeader,
+        args: List[Any],
+        *,
+        dst_addressing: Optional[
+            Union[t.Addressing.Group, t.Addressing.IEEE, t.Addressing.NWK]
+        ] = None,
+    ):
+        """Handle a cluster request."""
+        _LOGGER.debug(
+            "%s: handle_cluster_request - Command: %s Data: %s",
+            self.name,
+            hdr.command_id,
+            args,
+        )
+        if hdr.command_id == 0x00:
+
+            button = BUTTONS.get(args[0])
+            press_type = PRESS_TYPES.get(args[1])
```

Comment:
If you don't expect the provided `PRESS_TYPE` to be anything other than the integers 1-6, maybe use `PRESS_TYPES[args.press_type]`.